### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by davidwessman**
Removing unused tables from database
Removing old views and controllers

### CODE REVIEW

File: db/migrate/20160421130636_clean_up_unused.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    drop_table :subcategories
+
+    drop_table :cafe_works
+    drop_table :cafe_work_councils
+
+    drop_table :roles
+  end
+
+  def down
+    raise :error
```
Comment (davidwessman) [inline db/migrate/20160421130636_clean_up_unused.rb]: Not sure if this is the right way

**Comment by davidwessman** [comment]
@navro @jforberg 
I changed this to be reversible, however any data would be lost (but is not used).

**Comment by jforberg** [comment]
I don't think it's very important to have reversible migrations. As you say, all data will be gone anyway so you will still be f*ed. We have backups running which can save us from data loss.

**Comment by navro** [comment]
+1

**Pull request merged**